website/docs/docs/dbt-versions/release-notes/72-Feb-2024/ide-github.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "New: Edit in primary branch in IDE"
```

Comment:
i dont mind new but is this an enhancement to an existing system?? it's an improved user experience 

```suggestion
title: "Enhancement: Edit in primary branch in IDE"
```